src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -628,12 +629,12 @@ MStatus PRTModifierAction::updateDynamicEnums() {
 				const wchar_t* const* stringArray = mGenerateAttrs->getStringArray(valuesAttr, &arr_length);
 
 				for (short enumIndex = 0; enumIndex < arr_length; enumIndex++) {
-					std::wstring currString = stringArray[enumIndex];
+					const std::wstring currString = stringArray[enumIndex];
 
 					// remove newlines from strings, because they break the maya UI
-					currString.erase(std::remove(currString.begin(), currString.end(), '\n'), currString.end());
+					const std::wstring cleanedString = std::regex_replace(currString, std::wregex(L"\r|\n"), L"");

```

Comment:
using substring and find_last_of instead of regex to avoid newline characters. The new behavior is in line with how CE currently handles dynamic enums with newline characters.